pkg/operator/ceph/operator.go

```diff
@@ -227,3 +235,65 @@ func (o *Operator) updateDrivers() error {
 	logger.Infof("successfully started Ceph CSI driver(s)")
 	return nil
 }
+
+func (o *Operator) setCSIParams() error {
+	var err error
+
+	csiEnableRBD, err := k8sutil.GetOperatorSetting(o.context.Clientset, cephController.OperatorSettingConfigMapName, "ROOK_CSI_ENABLE_RBD", "true")
+	if err != nil {
+		return errors.Wrap(err, "unable to determine if CSI driver for RBD is enabled.")
```

Comment:
```suggestion
		return errors.Wrap(err, "unable to determine if CSI driver for RBD is enabled")
```

That's probably better this way but I can live without, if you have another change to push you can fix otherwise we can leave it like this.

Reply:
No new changes planned, unless the tests need some fixing.